src/server.rs

```diff
@@ -128,11 +128,14 @@ impl Server {
                 loop {
                     if let Some(event) = sqs_handler.fetch() {
                         if let Err(e) = Server::process_message(&db_pool, &event) {
+                            println!("SQS: Error: Could not process message: {:?}", e);
                             slog_error!(sq_logger.log, "Could not process message"; "error" => e.to_string());
                         };
+                        /*
```

Comment:
Yes, if this works, I'll pull it once I work out what's going on with slog in production and pull the printlns.